app/helpers/dashboard_helper.rb

```diff
@@ -40,10 +40,4 @@ def percentage_string(percentage)
 
     "#{percentage.round(0)}%"
   end
-
-  def percentage(numerator, denominator)
-    return percentage_string(0) if denominator.zero?
-
-    percentage_string((numerator * 100.0) / denominator)
-  end
```

Comment:
I see controllers like [this one](https://github.com/simpledotorg/simple-server/blob/master/app/controllers/api/current/analytics/user_analytics_controller.rb#L2) that only use the `DashboardHelper`. If we remove this method, won't we break those controllers?

Reply:
No, the `percentage` method is only used within MyFacilities.